<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterfaceSettings</class>
 <widget class="QWidget" name="UserInterfaceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="VerticalOnlyScroller" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>658</width>
        <height>790</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
       <property name="spacing">
        <number>32</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="generalGroup">
         <property name="title">
          <string>General Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="leftMargin">
               <number>40</number>
              </property>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="statusChanges">
              <property name="text">
               <string>Show contacts' status changes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="groupJoinLeaveMessages">
              <property name="text">
               <string>Add a chat message when a user joins or leaves a group</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="autoAwayLayout">
            <item>
             <widget class="QLabel" name="autoAwayLabel">
              <property name="toolTip">
               <string>Your status is changed to Away after set period of inactivity.</string>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Auto away after (0 to disable):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autoAwaySpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Set to 0 to disable</string>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>true</bool>
              </property>
              <property name="suffix">
               <string notr="true"> min</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Default directory to save files:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="autoSaveFilesDir">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Set where files will be saved.</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="autoacceptFiles">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string comment="autoaccept cb tooltip">You can set this on a per-friend basis by right clicking them.</string>
              </property>
              <property name="text">
               <string>Autoaccept files</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="maxAutoAcceptSizeLabel">
              <property name="text">
               <string>Max autoaccept file size (0 to disable):</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="maxAutoAcceptSizeMB">
              <property name="suffix">
               <string> MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalOnlyScroller</class>
   <extends>QScrollArea</extends>
   <header>src/UI/tool/verticalonlyscroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
